server/use-router.js

```diff
@@ -19,3 +23,4 @@ export default function useRouter({ res, req, routes, initialState }) {
     res.status(404).send('Not found');
```

Comment:
Another potential place to render 500 page is line 18 https://github.com/redbadger/react.london/pull/223/files#diff-ec0a57d23ecdabfd1e3675d3eaf8e3baR18 - when renderToString fails. This obv should never pass tests, but we could probably use that for dev purposes, and print out the React error message.